woocommerce/woocommerce-gutenberg-products-block
# Fix: Mini Cart block isn't available in the Customizer > Widget

Fixes #6188

This PR registers the Mini Cart block for Customizer, so classic themes can insert and edit the Mini Cart block in widget areas.

### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Activate the Storefront theme.
2. Go to Customizer > Widgets > Sidebar.
3. See Mini Cart can be added to the Sidebar.
4. Publish changes. See the block works on the front end.
 
### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
